plugins/automation/config/locales/client.en.yml

```diff
@@ -156,6 +156,9 @@ en:
             restricted_group:
               label: Group
               description: Optional, will trigger only if the post's topic is a private message in this group's inbox
+            restricted_groups:
+              label: Groups
+              description: Optional, will trigger only if the post's topic is a private message in this groups inbox
```

Comment:
```suggestion
              description: Optional, will trigger only if the post's topic is a private message in this group's inbox
```

Reply:
I think actually correct answer here is: `groups' inbox`

Reply:
`these groups'`?

Reply:
Made the copy clearer after checking with david